scripts/src/test-raffle.ts

```diff
@@ -493,9 +493,9 @@ async function single_test1(factoryContract: string) {
   await wait(6000);
   await claim(protocolWallet, airdropContract, [4000000]);
   await wait(6000);
-  await claim(hallwallet, airdropContract, [1000000])
-    .then(throwErr)
-    .catch(expectError("Error is thrown for being unable to claim"));
+  await claim(hallwallet, airdropContract, [1000000]);
+  // .then(throwErr)
+  // .catch(expectError("Error is thrown for being unable to claim"));
```

Comment:
not sure why this is commented out, and then the entire `single_test1` call is commented out anyway